platform/nephos/one-aboot.mk

```diff
@@ -0,0 +1,2 @@
+# sonic broadcom one image installer
```

Comment:
if only .bin is created, this file is not needed.

Reply:
thanks, I will update it.